src/applications/vaos/new-appointment/components/VAFacilityPage/EligibilityModal.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import { FETCH_STATUS } from '../../../utils/constants';
+import FacilityAddress from '../../../components/FacilityAddress';
+
+export default function EligibilityModal({
```

Comment:
This is a recreation of `<EligibilityCheckMessage />`, but with the `<AlertBox />` removed since we are displaying it within a modal.  It displays the different messaging we give when the user is not eligible to schedule at a selected clinic.